pkg/jobrunaggregator/jobrunbigqueryloader/disruption.go

```diff
@@ -163,17 +163,21 @@ func (o *disruptionUploader) uploadContent(ctx context.Context, jobRun jobrunagg
 		return nil
 	}
 
-	return o.uploadBackendDisruptionFromDirectData(ctx, jobRun.GetJobRunID(), backendDisruptionData, logger)
+	clusterData, _ := jobRun.GetOpenShiftTestsFilesWithPrefix(ctx, "cluster-data")
```

Comment:
Looks like another error that should be logged.

Reply:
Added, don't want to fail the uploading of disruption though so it will proceed even on error.